packages/frameworks/src/frameworks.ts

```diff
@@ -1942,6 +1964,11 @@ export const frameworks = [
     useRuntime: { src: 'package.json', use: '@vercel/hydrogen' },
     detectors: {
       some: [
+        {
+          path: 'package.json',
+          matchContent:
+            '"(dev)?(d|D)ependencies":\\s*{[^}]*"@shopify/hydrogen":\\s*".+?"[^}]*}',
+        },
```

Comment:
@TooTallNate was there a reason this looked for the config file and not the package? Do you think it's OK to add the package as an optional lookup as well?

https://github.com/vercel/vercel/pull/8073

Reply:
I don't recall, but package name lookup would probably be fine.